backend/feedback/index.js

```diff
@@ -1,9 +1,25 @@
 const fetch = require("node-fetch");
 require("dotenv").config();
 
-module.exports = async function (context, req) {

```

Comment:
Mostly so the context.done() would go into effect so that we can set it to actually stop running and also to match how the other azure function is set up— with the setup function, there was some weird behavior when it used to be async.